### PR TITLE
add internal API specs

### PR DESCRIPTION
**Pull request by wangkirin**
Signed-off-by: Wang Qilin <qilin.wang@huawei.com>

### CODE REVIEW

**Comment by genedna** [comment]
Should be like '/v1/rc' .

File: docs/specs/Internal.md [hunk 1]
```diff
@@ -1 +1,646 @@
...
+# Specification of internal APIs (version v1.0)
+
+## APIs of Registration Center
+
+### List
+
+|Method|Path|Request Source|Entity|Description|
+|------|----|--------------|------|-----------|
+|POST|`/internal/v1/dataserver`|Administrator|Data Server|Register a new data server or a batch of data servers|
+|DELETE|`/internal/v1/:dataserverID`|Administrator|Data Server|Delete a data server|
```
Comment (genedna) [inline docs/specs/Internal.md]: `:dataserver` is just fine.

File: docs/specs/Internal.md [hunk 2]
```diff
@@ -1 +1,646 @@
...
+|GET|`/internal/v1/:dataserverID`|Administrator|Data Server|Get data server info by a given data server ID|
+|PUT|`/internal/v1/dataserver`|Data Groups|Data Server|Recieve status from data server|
+|GET|`/internal/v1/groups/:groupID`|Object Server|Group|Get group info by a given groupID |
+|GET|`/internal/v1/groups`|Object Server|Group|Get all available groups for Object Server|
+|GET|`/internal/v1/object/id`|Object Server|Object|Returns a FileID allocated by Registration Center|
+|PUT|`/internal/v1/object`|Object Server|Object|Save/Update object info|
+|GET|`/internal/v1/object/:objectID`|Object Server|Object|Query object info|
```
Comment (genedna) [inline docs/specs/Internal.md]: `:object` is just fine.

File: docs/specs/Internal.md [hunk 3]
```diff
@@ -1 +1,646 @@
...
+
+### List
+
+|Method|Path|Request Source|Entity|Description|
+|------|----|--------------|------|-----------|
+|POST|`/internal/v1/dataserver`|Administrator|Data Server|Register a new data server or a batch of data servers|
+|DELETE|`/internal/v1/:dataserverID`|Administrator|Data Server|Delete a data server|
+|GET|`/internal/v1/:dataserverID`|Administrator|Data Server|Get data server info by a given data server ID|
+|PUT|`/internal/v1/dataserver`|Data Groups|Data Server|Recieve status from data server|
+|GET|`/internal/v1/groups/:groupID`|Object Server|Group|Get group info by a given groupID |
```
Comment (genedna) [inline docs/specs/Internal.md]: `:group` is just fine.

File: docs/specs/Internal.md [hunk 4]
```diff
@@ -1 +1,646 @@
...
+Content-Type: application/json
+```
+```json
+[
+  {
+    "groupID": 1,
+    "ip": "10.1.0.6",
+    "port": 7654
+  },
+  ……
```
Comment (genedna) [inline docs/specs/Internal.md]: JSON format is broken

**Comment by genedna** [comment]
@wangkirin The parameter name of REST API should be simplest. Please update it and correct JSON format in all returns.

**Comment by wangkirin** [comment]
@genedna  Already fixed!

**Comment by genedna** [comment]
OK, merge #4.

**Pull request merged**